packages/site/src/layout/SearchBox.tsx

```diff
@@ -74,34 +84,60 @@ export const SearchBox = ({
           onChange={d => setSearch(d as string)}
         />
         <div className={styles.searchBoxResults}>
-          {filteredComponentList.length > 0 && (
-            <Content>
-              <Typography
-                size={"base"}
-                fontWeight={"bold"}
-                textCase={"uppercase"}
-                textColor={"textSecondary"}
-                element="h3"
-              >
-                Components
-              </Typography>
-              <ContentCardWrapper>
-                {filteredComponentList.map(({ title, to, imageURL }, key) => {
-                  return (
-                    <ContentCard
-                      onClick={closeModal}
-                      title={title}
-                      to={to}
-                      imageURL={imageURL}
-                      key={key}
-                    />
-                  );
-                })}
-              </ContentCardWrapper>
-            </Content>
-          )}
-          {filteredDesignList.length > 0 && (
-            <Box padding={{ top: "largest" }}>
+          <Content spacing={"larger"}>
```

Comment:
I know Rebecca added the same change to her styles PR that isn't in yet. Will coordinate with her if end up merging first

Reply:
Looks like it got dealt with in the merge conflict! 👍🏻